### PR TITLE
ARM build configuration

### PR DESCRIPTION
**Pull request by GuyAv46**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -115,7 +123,7 @@ commands:
           name: Flow tests
           command: |
             python3 -m pip install -r sbin/requirements.txt
-            python3 -m tox -e flowenv
+            python3 -m tox -e flowenv -r
```
Comment (DvirDukhan) [inline .circleci/config.yml:126]: Do we need this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -59,9 +59,17 @@ commands:
       - run:
           name: basic install
           command: |
-            apt-get update
+            apt-get update --fix-missing
```
Comment (DvirDukhan) [inline .circleci/config.yml:62]: Do we need this?

File: pyproject.toml [hunk 3]
```diff
@@ -28,15 +28,15 @@ classifiers = [
 python = "^3.7"
 
 [tool.poetry.dev-dependencies]
-numpy = "1.21"
-tox = "^3.24.1"
+numpy = "^1.21"
```
Comment (DvirDukhan) [inline pyproject.toml:31]: If arm runs tiny build, do we need the modifications in this file and the requirements file?

File: sbin/system-setup.py [hunk 4]
```diff
@@ -13,40 +13,42 @@
...
-
     def debian_compat(self):
         self.run("%s/bin/getgcc --modern" % READIES)
-        self.install("python3-dev clang-format valgrind")
+        self.install("python3-dev valgrind")
 
     def redhat_compat(self):
         self.install("redhat-lsb-core")
+        self.run("%s/bin/getepel" % READIES, sudo=True)
+        self.install("libatomic")
```
Comment (DvirDukhan) [inline sbin/system-setup.py:32]: Why the additions here?

File: sbin/system-setup.py [hunk 5]
```diff
@@ -13,40 +13,42 @@
...
+
         self.run("%s/bin/getgcc --modern" % READIES)
 
     def fedora(self):
+        self.install("libatomic")
```
Comment (DvirDukhan) [inline sbin/system-setup.py:37]: Why the addition here?

File: .circleci/config.yml (hunk 1 above)
Comment (GuyAv46) [reply .circleci/config.yml:126]: No.

File: pyproject.toml (hunk 3 above)
Comment (GuyAv46) [reply pyproject.toml:31]: No, but why not upgrade?

File: sbin/system-setup.py (hunk 5 above)
Comment (GuyAv46) [reply sbin/system-setup.py:37]: the entire file was updated

File: .circleci/config.yml (hunk 2 above)
Comment (chayim) [reply .circleci/config.yml:62]: Sometimes apt loses its mind during updates. An update with --fix-missing solves many of these issues, when possible It rebuilds the index

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**